app/components/autocomplete/autocomplete_section_header.js

```diff
@@ -25,13 +25,12 @@ export default class AutocompleteSectionHeader extends PureComponent {
     render() {
         const {defaultMessage, id, loading, theme, isLandscape} = this.props;
         const style = getStyleFromTheme(theme);
-
         return (
             <View style={style.sectionWrapper}>
                 <View style={[style.section, padding(isLandscape)]}>
                     <FormattedText
-                        id={id}
-                        defaultMessage={defaultMessage}
+                        id={id.toUpperCase()}
```

Comment:
why toUpperCase?

Reply:
The original intent was to make the section header text uppercase as the design specs required. I am not quite sure why I thought putting a `toUpperCase()` there would be a good idea

I replaced this with using `textTransform: 'uppercase'` instead!